src/main_loop/proof_upgrader.rs

```diff
@@ -422,6 +426,39 @@ impl UpgradeJob {
         }
     }
 
+    fn gobbler_notification_method_with_receiver_preimage(
+        own_wallet_entropy: &WalletEntropy,
+        notification_policy: FeeNotificationPolicy,
+    ) -> (UtxoNotifyMethod, Digest) {
+        let gobble_receiver_key = match notification_policy {
```

Comment:
The type here is `SpendingKey`. But variable name suggests an address "receiver_key". How about `gobble_beneficiary_key`?